eventing/samples/github-source/subscription.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Subscription
+metadata:
+  name: github-source-sample
+  namespace: knative-demo
+spec:
+  channel:
+    apiVersion: eventing.knative.dev/v1alpha1
+    kind: Channel
+    name: githubchannel
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1alpha1
+      kind: Service
+      name: github-message-dumper
+
+---
+# This is a very simple Knative Service that writes the input request to its log.
+
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: github-message-dumper
+  namespace: knative-demo
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: github.com/knative/eventing-sources/cmd/message_dumper
```

Comment:
we could push that to `gcr.io` or `docker.io` - that should avoid the `ko apply`, right ?

Reply:
Actually, no reason to hold this up for that. The other samples also use `ko apply` here.

Reply:
I'm fine with putting in a TODO, but we'd like to have our samples not depend on `ko` and work cleanly with `kubectl apply` (possibly including using build to generate the container).

Reply:
I created #541 to track this, since all eventing samples have the same issue.